_posts/2015-10-28-refugeehackathon.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "#refugeehackathon"
+date: "2015-10-28 07:00:00"
+icon: flag
+categories: hackathon
+authors: ["Jan"]
+---
+
+Lots of refugees arrived in Germany.
+Task like organising accomodations, food and clothing or supporting them with their onwards journey raises many challenges.
+Though I am not able to work day and night for the solution like some people in the crisis management group, I wanted to fly the flag to show that a significant amount of people are willing to help.
+This blog post is a report about participating in the #refugeehackathon in Berlin, Germany.
+
+# Overview of the event
+
+The [#refuggeehackathon](https://twitter.com/search?src=typd&q=%23refugeehackathon) started on Friday, 24 October 2015 with a requirements workshop.
+A couple of people visited the refugees and talked about project ideas which could be helpful for them.
```

Comment:
A small group of people visited the refugees and talked about project ideas which could be helpful for them.